### PR TITLE
Some fixes for wix4 tests

### PR DESCRIPTION
**Pull request by barnson**

### CODE REVIEW

File: test/src/WixTests/Integration/BuildingPackages/Components.ComponentTests.cs [hunk 1]
```diff
@@ -82,7 +82,7 @@ public void CircularReferences()
             light.Run();
         }
 
-        [NamedFact]
+        [NamedFact(Skip = "Ignored because currently the compiler crashes for child elements with an invalid id.")]
```
Comment (rseanhall) [inline test/src/WixTests/Integration/BuildingPackages/Components.ComponentTests.cs:85]: So these tests weren't fixed by the next commit?

Comment (barnson) [reply test/src/WixTests/Integration/BuildingPackages/Components.ComponentTests.cs:85]: Nope, not sure of the right fix, so it's https://github.com/wixtoolset/issues/issues/5464.

File: test/data/Burn/DependencyTests/A.wxs [hunk 2]
```diff
@@ -11,7 +11,7 @@
 <?define TestName = Test?>
 <?endif?>
 
-<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:dep="http://schemas.microsoft.com/wix/DependencyExtension">
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:dep="http://wixtoolset.org/schemas/v4/wxs">
```
Comment (robmen) [inline test/data/Burn/DependencyTests/A.wxs]: Is DependencyExtension really in the root namespace? If so, that is a(nother) bug.

Comment (barnson) [reply test/data/Burn/DependencyTests/A.wxs]: It's possible that while I was lovingly and carefully hand-editing the files, I made a mistake. 15 times.

**Comment by barnson** [comment]
Down to 203 failures.

**Comment by robmen** [comment]
I tried to bring this in but if I cleaned my enlistment (no `build` or `packages` folder) then the build fails looking for xunit 2.1.0. Not exactly sure which way to fix it.

**Comment by barnson** [comment]
Which project? The VS test runner is supposed to handle 1.9.2.

**Comment by robmen** [comment]
`test\src\WixTests\WixTests.csproj`

**Comment by barnson** [comment]
Better now.

**Comment by robmen** [comment]
Build failure resolved, but now it looks like release build is running all the tests by default. This is great except that many still fail (and the tests take ~15min which is a different problem to solve).

Did you enable running test in Release by default? (Sorry, I didn't go searching through the 684 touched files to find it...)

**Comment by barnson** [comment]
Yeah, I disabled that but didn't commit. Fixed.

**Pull request merged**